src/main/webapp/WEB-INF/views/admin/project/list-js.jsp

```diff
@@ -28,7 +30,12 @@
 		//}
 		
 		showHelpLink("Project_List_Main");
-	});	
+
+
+		//ㅊㄱ
```

Comment:
In my opinion, it's better to remove unnecessary comments.

Reply:
https://github.com/fosslight/fosslight/commit/bab4b013a24ca57c981d332f09c1f0014d313359